packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -68,6 +68,310 @@ export const studioLocaleStrings = {
+
+  /** Label for "all fields" in the filter menu */
+  'navbar.search.all-fields': 'All fields',
+
+  /** Label for "shared fields" in the filter menu */
+  'navbar.search.shared-fields': 'Shared fields',
+
+  /** Label for "unknown types" in the filter menu */
+  'navbar.search.unknown-type-label': 'Unknown type',
+
+  /** Label for boolean filter - true */
+  'navbar.search.true': 'True',
+
+  /** Label for boolean filter - false */
+  'navbar.search.false': 'False',
+
+  /** Label for number and string filter */
+  'navbar.search.value': 'Value',
+
+  /** Label for number range filter - min placeholder */
+  'navbar.search.min-value': 'Min value',
+
+  /** Label for number range filter - max placeholder */
+  'navbar.search.max-value': 'Max value',
+
+  /** Label for search value in a range of numbers */
+  'navbar.search.number-items-range': `{{min}} → {{max}} items`,
+
+  /** Title label for when no search results are found */
+  'navbar.search.no-results-title': 'No results found',
+
+  /** Description for when no search results are found */
+  'navbar.search.no-results-description': 'Try another keyword or adjust your filters',
+
+  /** Title label for when the search has found an error */
+  'navbar.search.error.something-went-wrong-title': 'Something went wrong while searching',
+
+  /** Description label for when the search has found an error */
+  'navbar.search.error.something-went-wrong-description':
+    'Please try again or check your connection',
+
+  /** Label for when the asset source is unknown (used in the nav bar search, specifically filter) */
+  'navbar.search.error.unknown-asset-source': 'Unknown asset source found',
+
+  /** Title for error when no valid asset sources found */
+  'navbar.search.error.no-valid-asset-source-title': 'No valid asset sources found.',
+
+  /** Description for error when no valid asset source is found, describes that only the default asset is supported */
+  'navbar.search.error.no-valid-asset-source-only-default-description':
+    'Currently, only the default asset source is supported.',
+
+  /** Description for error when no valid asset source is found, describes that you should cehck the the current studio config */
+  'navbar.search.error.no-valid-asset-source-check-config-description': `Please ensure it's enabled in your studio configuration file.`,
+
+  /** Title for error when a filter cannot be displayed */
+  'navbar.search.error.display-filter-title': 'An error occurred whilst displaying this filter.',
+
+  /** Description for error when a filter cannot be displayed, describes that you should check the schema */
+  'navbar.search.error.display-filter-description':
+    'This may indicate invalid options defined in your schema.',
+
+  /** Label for error when the useSearchState is not used within the SearchProvider */
+  'navbar.search.error.use-search-state-not-used-within-provider':
+    'useSearchState must be used within an SearchProvider',
```

Comment:
Ah this is a good point that I din't consider 🤔 
I think we probably need to but probably not at the client level (because you wouldn't want typos on specific hooks for example). But @rexxars what are your thoughts on this?